po/uk.po

```diff
@@ -366,12 +366,12 @@ msgstr "Закрийте вікно, щоб вийти..."
 #: src/Gtk/MainWindow.vala:391
 #, fuzzy
 msgid "Uninstall Old"
-msgstr "Видалення завершено"
+msgstr "Видалити встановлені ядра старіши за встановлене"
```

Comment:
IMO there's a translation mistake `старіши` should be `старші`

Reply:
Also it would be better not to extend the message. `Uninstall Old` (line 368) would be more like `Видалити старі` (line 369). Afterwards fuzzy flag may be removed